openquake/mbt/tools/faults.py

```diff
@@ -50,25 +50,53 @@ def get_fault_vertices_3d(fault_trace, upper_seismogenic_depth,
 
     return all_lons, all_lats, all_deps
 
+def get_rate_above_m_cli(mma, rrr, m_min, m_cli, bin_width):
+    """
+    :parameter mma:
+        A list containing the rates per bin starting from m_min
+    :parameter rrr:
+        A list containing the magnitude bins starting from m_min
+    :parameter m_min:
+        Minimum magnitude
+        float
+    :parameter m_cli:
+        Clipping magnitude
+        float
+    :parameter bin_width:
+        Bin width
+    :return:
+        A list containing the rates per bin starting from m_cli
+    """
+    if m_cli+bin_width/2. == m_min+bin_width/2.:
+        rate_m_cli = rrr
 
-def _get_rate_above_m_low(seismic_moment, m_low, m_upp, b_gr, a_m=9.05):
+        return rate_m_cli
```

Comment:
Here you can remove the previous line and just write `return rrr`

Reply:
Ok, I agree